### PR TITLE
refactor release access and drop cluster-service dependency

### PR DESCRIPTION
**Pull request by xh3b4sd**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: flag/service/service.go [hunk 1]
```diff
@@ -12,10 +11,9 @@ import (
 
 // Service is an intermediate data structure for command line configuration flags.
 type Service struct {
-	ClusterService clusterservice.ClusterService
```
Comment (xh3b4sd) [inline flag/service/service.go:15]: We don't need this anymore.

File: service/controller/resource/releaseversions/resource.go [hunk 2]
```diff
@@ -69,32 +69,22 @@ func (r *Resource) ensure(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
-	var versionBundles []versionbundle.Bundle
-	var apps []versionbundle.App
+	var re releasev1alpha1.Release
```
Comment (xh3b4sd) [inline service/controller/resource/releaseversions/resource.go:72]: Fetching the `Release` CR now using the controller client.

**Comment by xh3b4sd** [comment]
Tested on `gauss`.

**Comment by tuommaki** [comment]
Nice! :rocket:

**Pull request merged**